package.json

```diff
@@ -35,22 +36,25 @@
     "file-loader": "3.0.1",
     "flow-bin": "0.54.1",
     "lqip-loader": "^2.2.0",
-    "webpack-dev-server": "^3.1.14"
+    "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
     "@babel/cli": "7.2.3",
     "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
-    "@rails/webpacker": "^4.0.2",
+    "@rails/webpacker": "^4.0.7",
     "babel-loader": "^8.0.5",
     "babel-plugin-flow-runtime": "0.11.1",
     "babel-plugin-jsx-control-statements": "3.2.8",
     "babel-plugin-transform-decorators-legacy": "1.3.4",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.23",
+    "@babel/preset-env": "^7.3.4",
```

Comment:
This preset is showing up as missing when running `make build`. Adding it makes sense, but I am still investigating exactly why I am now seeing this error. Likely it could be due to https://github.com/powerhome/playbook/pull/103/files#r310702140